tests/Components/test_BaseComponent.py

```diff
@@ -52,6 +52,14 @@ def payload():
     )
     return payload
 
+@pytest.fixture
+def moment1():
+    mag = 25
+    ang = math.radians(30)
+    moment_point = Vector2(-5,2)
+    moment1 = Vector2(r=mag,theta=ang)
+    return moment1, moment_point
+
```

Comment:
Moments do not have a point of application, physically speaking, so you can remove *moment_point* instantiation and return.

About the moment being a vector, it is not physically wrong, but ADR currently deals with 2D physics only, so we consider the moment a scalar (just having magnitude) and always assume that it acts on the Z-axis (pitch). On future versions we will probably introduce 3D and consider moments as vectors.